docs/getting_started/basics/batch.md

```diff
@@ -0,0 +1,35 @@
+# Batch Execution
+
+Shift is able to run workflows without the need of a UI interface. This is called Batch mode and can be used via the Shift Python API.
```

Comment:
Just a question, I have seen different ways when we mention Python, in some cases (different pages) is written as `Python` and in other cases as `python`, it's a little bit inconsistent.
Can we decide which is the best way to keep consistency between the different pages? @Inbibo/shift